### DISCUSSION TITLE
Should language config use YAML instead of JSON?

### DISCUSSION THREAD

**Discussion by MrGriefs** [just-a-paw/language#14] [_Misc]
This project intends to be welcoming to developers and non-developers, and translators should create entries with ease.  
JSON, in my opinion, can be too coarse and intimidating to translators willing to work on this project without prior experience with programming.
The use of YAML ([spec](https://yaml.org/spec/1.2.2)) in this project, where translators interact, may bring the project closer to availability.

**Comment by RamonvRooyen** [comment]
In my opinion JSON has better readability and is more forgiving. YAML has some weird conventions that are not immediatly apparant. (eg. multiline) which makes it more prone to errors in my opinion

**Comment by CrystiqDev** [comment]
Personally i would use JSON because you a better overview on everything, YAML would not be a wrong choice but it is messier. Therefore i would choice JSON

**Comment by JulianWogersien** [comment]
In my opinion JSON would be the better choice because of better readability

**Comment by ghost** [comment]
JSON is already a YAML-subset IIRC :-)

I personally have no problem with the JSON sources, but I might be biased. The fact that translations are submitted through GitHub may render the issue of friendliness at least somewhat moot simply because those who use GitHub are probably going to be more technically inclined – say programmers or those who are otherwise comfortable with e.g. JSON, version-control, etc. I could be wrong though; this is just conjecture.

**Comment by FurieFox** [comment]
I think you should stay with JSON and maybe train a bit the new translators, so they can be used to it. If they need some help, they could still ask you anything they would need in order to work.
